game.py

```diff
@@ -0,0 +1,98 @@
+# Here is the description of the game simulation.
+
+# this is a two player card game
+# the game starts with a deck of cards
+# the cards are dealt out to both players
+# on each turn:
+# both players turn over their top-most card
+# the player with the higher valued card takes the cards and puts them in their scoring pile (scoring 1 point per card)
+# this continues until the players have no cards left
+# the player with the highest score wins
+# It’s considered a simulation because the players don’t have any choice, dont worry about input
+import random
+
+class CardDeck():
+    suits = ['spade', 'heart', 'diamond', 'club']
+    vals = [i for i in range(1,14)]
+    def __init__(self):
+        self.cards = [Card(suit, val) for suit in self.suits for val in self.vals]
+        random.shuffle(self.cards)
+
+class Card:
+    def __init__(self, s, v): # card: (type, val)
+        self.suit = s
+        self.val = v
+    def __gt__(self, other):
+        if self.val > other.val:
+            return True
+        if self.val == other.val:
+            if self.suit == 'spade':
+                return True
+            if self.suit == 'heart' and other.suit != 'spade':
+                return True
+            if self.suit == 'diamond' and other.suit == 'club':
+                return True
+            else:
+                return False
+        return False
+    
+    def __lt__(self, other):
+        if self.val < other.val:
+            return True
+        if self.val == other.val:
+            if other.suit == 'spade':
+                return True
+            if other.suit == 'heart' and self.suit != 'spade':
+                return True
+            if other.suit == 'diamond' and self.suit == 'club':
+                return True
+            else:
+                return False
+        return False
+
+class Player:
+    def __init__(self):
+        self.cards = []
+        self.score = 0
+
+class Game:
+    def __init__(self):
+        self.player1 = Player()
+        self.player2 = Player()
+        self.cards = CardDeck().cards
```

Comment:
Here you create a CardDeck() object but you didn't create a variable to store it. This is unsafe because it's not guaranteed if this will be cleaned up by Python.
You should initialize CardDeck object here and then access its `cards` after that:
```
self.card_deck = cardDeck()
self.cards = self.card_deck.cards
```